src/App.vue

```diff
@@ -336,7 +332,8 @@ export type Shader = {
     layout: Bindings,
     hashedStrings: any,
     reflection: ReflectionJSON,
-    threadGroupSize: ThreadGroupSize | { x: number, y: number, z: number },
+    threadGroupSize: { [key: string]: ThreadGroupSize },
+    entryPoint: string,
```

Comment:
Why are you adding `entryPoint` to the shader when the goal of this pr is to allow multiple entrypoints in one shader? At minimum it seems this should be called `mainEntrypoint`.